server/gitlab/api.go

```diff
@@ -321,37 +339,64 @@ func (g *gitlab) GetYourPrs(ctx context.Context, user *UserInfo) ([]*internGitla
 		return nil, err
 	}
 
-	return result, nil
+	var resultWithLabelDetails []*MergeRequest
```

Comment:
I said it to change everywhere.
```suggestion
	var mergeRequests []*MergeRequest
```